cli/src/tests.rs

```diff
@@ -253,3 +253,58 @@ async fn cancel_invitation(tmp_path: TmpPath) {
             "Deleting invitation\nInvitation deleted",
         ));
 }
+
+#[rstest::rstest]
+#[tokio::test]
+async fn list_invitations(tmp_path: TmpPath) {
```

Comment:
Is it possible to have the other cases tested (`Deleted` & `Ready`) ?

Reply:
I don't know how to reproduce them

Reply:
If you delete the `invitation`, it will be absent instead of the status `deleted`